packages/paste-codemods/tools/.cache/mappings.json

```diff
@@ -21,6 +21,7 @@
   "ChatAttachment": "@twilio-paste/core/chat-log",
   "ChatAttachmentDescription": "@twilio-paste/core/chat-log",
   "ChatAttachmentLink": "@twilio-paste/core/chat-log",
+  "ChatBookend": "@twilio-paste/core/chat-log",
```

Comment:
thought (blocking): you may have to re-run the build to update this with ChatEvent

Reply:
Oh good catch, thanks! Running it now